blocks/rich-text/index.js

```diff
@@ -375,15 +375,33 @@ export class RichText extends Component {
 	 * Handles any case where the content of the tinyMCE instance has changed.
 	 */
 
-	onChange() {
+	onChange( options ) {
+		const { checkEquality = false } = options;
+
 		this.savedContent = this.getContent();
+
+		if ( checkEquality && isEqual( this.props.value, this.savedContent ) ) {
```

Comment:
Curious when does this thing happen. I mean `this.props.value` and `this.savedContent` being equal but not strictly equal. this.savedContent was introduced exactly to avoid this kind of changes, so I'm wondering if we're missing a `this.savedContent` assignment somewhere?

Reply:
Hm, that's in the updateContent function to avoid setting the content. This function does not do a strict equal check. `isEqual( this.props.value, this.savedContent )` will be true when the content is in sync, e.g. blurring the field.

Reply:
>`isEqual( this.props.value, this.savedContent )` will be true when the content is in sync, e.g. blurring the field.

When and why is this causing `onChange` to be called? Doesn't seem like it should be.

Reply:
I changed it. It was to prevent dispatching actions when the content is the same (coming from a TinyMCE change event). This only happens on blur, so we can filter that one out.